### ISSUE TITLE
Suggestion only - Meanings

### ISSUE THREAD

**Issue by MikeBound** [A-mIn3/WINspect#6]
Might not be right place to drop this :)
What do I mean about "meanings".
Some notes somewhere that says, that if you get results in group, what it means and recommendation.

Example 1:
[?] Checking for configurable services ..
Name                                                                                            Path                                                                                          
----                                                                                            ----
NetTcpPortSharing                                                                               C:\WINDOWS\Microsoft.NET\Framework64\v4.0.30319\SMSvcHost.exe     

Example 2:
[?] Checking for unquoted path services ..
Name         Sate         Start Mode   Path                                                                                                                                                   
----         ----         ----------   ----                              
AGSService   Running      Auto         C:\Program Files (x86)\Common Files\Adobe\AdobeGCClient\AGSService.exe    

There'll be a reason why they are a problem and should be investigated.
Note specifics on each item found is not required. But what each section is trying to highlight as being the problem.
Somes sections like autoruns, are more obvious.
I hope clear.

**Comment by A-mIn3** [comment]
I can't really get what you want to say.
I think you are asking about the reason why we are checking for unquoted path services, you can refer to resources explaining the vulnerability on the web.
KR.

**Comment by MikeBound** [comment]
Sorry. I've edited/fixed the writing above.
You are right, once you search about "unquoted path services", there is info you can find.
Now trying to find info on "configurable services". _This is not so easy to find info on_. Do you have any suggestion of a reference to this issue? Just asking :)

**Comment by A-mIn3** [comment]
In the context of an authenticated, unprivileged user, being able to configure a particular service (i.e changing the binary image of the service executable for example) can allow the user to replace the executable with a malicious program which will run in the security context of the service, usually LocalSystem.

**Comment by A-mIn3** [comment]
In the context of an authenticated, unprivileged user, being able to configure a particular service (i.e changing the binary image of the service executable) can allow the user to replace the executable with a malicious program which will run in the security context of the service, usually LocalSystem.

**Comment by MikeBound** [comment]
Thank you for taking the time to respond.
We reviewed your script to see what it was doing. So we could get a handle on what it meant.
I had concluded it meant what you described above. I think pretty cool to be aware of this.
The .net thing above has the following outcome and why it popped.
CR = SERVICE_USER_DEFINED_CONTROL – send a service control defined by the service’s authors.
Am not sure what status is for other people, but for us the service is in a disabled start mode.
Maybe a CR status is bad. But am struggling to find info that explains what it means. Plenty how to configure it stuff. Sort of assuming the service control is just start, stop and pause? And that CR is action on these things, defined by the author of the service.
Just trying to give food for thought really. Like what I'm seeing on this so far :)
